package.json

```diff
@@ -47,13 +47,13 @@
       "js"
     ],
     "testRegex": "/src/.*\\.test\\.(ts|tsx)$",
-    "testEnvironment": "node",
+    "testEnvironment": "jsdom",
     "moduleNameMapper": {
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "\\.(css|scss|less)$": "identity-obj-proxy",
       "^react$": "preact-compat",
       "^react-dom$": "preact-compat",
-      "@(.*)$": "<rootDir>/src/$1"
+      "@/(.*)$": "<rootDir>/src/$1"
```

Comment:
why did you added the slash (/) here?

Reply:
I did because without this raised an error in test case, i think this is occured, because i try to call the component like this: 
`import X from '@x'`

insteed this: 
`import X from '@/X'`

I tested now without the slash and still working, so i can remove in the next commit.

Reply:
nice, thank you very much mate =)